buildout.cfg

```diff
@@ -1,11 +1,13 @@
 [buildout]
+extensions = mr.developer
```

Comment:
we don't really need mr.developer if no checkouts are done, right? with the ``develop = .`` is enough to use this package in development mode.